contracts/Deployer.sol

```diff
@@ -49,6 +61,7 @@ contract Deployer is Destructible, GroupAdmin {
         uint _coolingPeriod,
         address _tokenAddress
     ) external {
+        ConferenceTicket ct = new ConferenceTicket(baseTokenUri);
```

Comment:
What's the benefit of `Deployer` instantiate `ConferenceTicket` contract and pass the address to `AbstractConference` instead of `Conference` creates `ConferenceTicket` on its constructor or `Conference` itself is ERC721 contract?

Reply:
That was my first thought as well to just inherit ERC721 in `AbstractConference`. Unfortunately, this makes the contract being too large; thus not deployable.

> Error: Transaction reverted: trying to deploy a contract whose code is too large

The same happens when trying to instantiate `ConferenceTicket` inside the constructor of `AbstractConference`.